shell/platform/windows/flutter_windows_engine.h

```diff
@@ -262,6 +262,22 @@ class FlutterWindowsEngine {
   // Registers the top level handler for the WM_CLOSE window message.
   void OnApplicationLifecycleEnabled();
 
+  // Called when a Window receives an event that may alter the application
+  // lifecycle state.
+  void OnWindowStateEvent(HWND hwnd, WindowStateEvent event);
+
+  WindowsLifecycleManager* GetLifecycleManager() {
```

Comment:
Nit: Our style is to name getter functions like variables

```suggestion
  WindowsLifecycleManager* lifecycle_manager() {
```